sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_call_connection_client.py

```diff
@@ -463,6 +467,7 @@ def play_media_to_all(
         loop: bool = False,
         operation_context: Optional[str] = None,
         operation_callback_url: Optional[str] = None,
+        interrupt_call_media_operation: Optional[bool] = None,
```

Comment:
as discussed offline, let's change this to default `False`

Reply:
setting default to false, fails other tests, as playoptions from generated model sets the interrupt_call_media_operation flag bool or none for the expected play request.  
![image](https://github.com/Azure/azure-sdk-for-python/assets/146493756/0c86a3ca-f748-493e-a75c-1ee148ad34dc)

Reply:
@Paresh-Arvind-Patil what happens in the backend if the `InterruptCallMediaOperation` flag is set to null? Does it even allow it?

Reply:
@v-vdharmaraj please test this SDK and provide what tests you ran and the results.
We may need to update the tests instead of allowing invalid values on our SDK.

Reply:
Hi @cochi2 , If the flag is set to null, it will be assigned a default value of boolen which is false. Also i agree the default options should be false as on the server side the flag is `bool InterruptCallMediaOperation` and not a nullable boolean `bool? InterruptCallMediaOperation`. 
This is same with as what we currently have with the Recognize `InterruptCallMediaOperation` flag

Reply:
@cochi2, I have updated to play_media_to_all to set the default value for the interrupt_call_media_operation. in either way server looks like takes false for null/false.

Reply:
> Thank you, @v-vdharmaraj. Please do the testing on Contoso app and report back your results.

YEs im trying to test, im facing some issues on the contoso app i think some of the dependencies (call automation) blocking contoso app to install the dependencies. So trying to create venv and test it. Will test the changes and will update here.

Reply:
> > Thank you, @v-vdharmaraj. Please do the testing on Contoso app and report back your results.
> 
> YEs im trying to test, im facing some issues on the contoso app i think some of the dependencies (call automation) blocking contoso app to install the dependencies. So trying to create venv and test it. Will test the changes and will update here.

@cochi2  Tested using Contoso app. Looks good to me and looks like currently its failing on the live test, I might need help on fixing those as i dont have access on the live test connection strings to test against